deploy/olm-catalog/qdr-operator/0.1.0/catalog-source.yaml

```diff
@@ -48,50 +49,88 @@ items:
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: TLS/SSL configuration for connections
-                  displayName: SslProfiles
-                  path: sslProfiles
+                  displayName: SslProfiles Name
+                  path: sslProfiles.name
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Address configuration for distribution and phasing
-                  displayName: Addresses
-                  path: addresses
+                  displayName: Addresse Prefix
+                  path: addresses.prefix
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Auto links address configuration
-                  displayName: AutoLinks
-                  path: autoLinks
+                  displayName: AutoLinks Address
+                  path: autoLinks.address
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Link routes address configuration
-                  displayName: LinkRoutes
-                  path: linkRoutes
+                  displayName: LinkRoute Prefix
+                  path: linkRoutes.prefix
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Outbound client connections to AMQP 1.0 entity
-                  displayName: Connectors
-                  path: connectors
+                  displayName: Connector Name
+                  path: connectors.Name
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Outbound interior router connections for mesh operation
-                  displayName: InterRouterConnectors
-                  path: interRouterConnectors
+                  displayName: InterRouterConnector Name
+                  path: interRouterConnectors.name
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 - description: Outbound edge router connections for edge uplink
-                  displayName: EdgeConnectors
-                  path: edgeConnectors
+                  displayName: EdgeConnector Name
+                  path: edgeConnectors.name
+                  x-descriptors:
+                    - "urn:alm:descriptor:com.tectonic.ui:label"
                 statusDescriptors:
                 - description: The current revision of the Interconnect cluster
                   displayName: Revision Number
                   path: revNumber
                 - description: The current pods
                   displayName: Pods
                   path: pods
-                - description: The current conditions
-                  displayName: Conditions
```

Comment:
I think the conditions are worth keeping (though I think the operator's use of them needs to be improved a little). There is an x-descriptor available that will display them.

Reply:
Condition added back. Please refer to the pic above